plasmapy/formulary/tests/test_densities.py

```diff
@@ -0,0 +1,33 @@
+"""Tests for functionality contained in `plasmapy.formulary.frequencies`."""
+import astropy.units as u
+import numpy as np
+import pytest
+
+from plasmapy.formulary.densities import plasma_critical_density
+from plasmapy.formulary.frequencies import plasma_frequency
+
+n_i = 5e19 * u.m**-3
+ion = "e-"
+
+
+def test_plasma_critical_density():
+    r"""Test the plasma_critical_density function in frequencies.py."""
+
+    omega_p = plasma_frequency(n=n_i, particle=ion)
+    n_c = plasma_critical_density(omega_p)
+
+    assert n_c.unit.is_equivalent(u.m**-3)
+
+    assert np.isclose(n_c, 5e19 * u.m**-3)
+
+    with pytest.raises(TypeError):
```

Comment:
I've asked a question about this on #1667 , but I don't think these need to be tested explicitly? If you do keep them, they should also be separate test functions. You can keep that organized by using pytest fixtures and parameterize (e.g. make one function to test errors, parameterized with a list of inputs and expected error types)

Reply:
I'll probably get rid of them